python/antworld/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools", "wheel", "scikit-build", "cmake", "cv2", "numpy"]
```

Comment:
why not just use setuptools? it's a bit shit but almost universal - I've never encountered scikit-build before...

Reply:
It's just like setuptools (I think) except it builds your CMake project for you beforehand, which is why I chose it.

Reply:
Ahh cool - that makes sense - setup tools does demand that it's in charge of building extensions